### ISSUE TITLE
Lack of .dbf in test cases

### ISSUE THREAD

**Issue by joshday** [JuliaGeo/Shapefile.jl#80]
I just realized all but one of the cases in `test/shapelib_testcases` do not have a .dbf file.  Given .dbf is mandatory according to the ESRI spec, we should add those.

In my quick search of http://shapelib.maptools.org/ I didn't see where those test cases are taken from.

**Comment by rafaqz** [comment]
It may be so long ago they are gone now/ @yeesian do you remember the source of the test cases here?

**Comment by yeesian** [comment]
Yeah sorry I don't fully remember what I did back then to generate them by now. A search on google turned this up: https://github.com/OSGeo/shapelib/blob/6189dbd83ba2382fe0ebbc8f2308e582832c93fa/shptest.c#L238-L275

From what I read in [`/test/shapelib_testcases/readme.txt`](https://github.com/JuliaGeo/Shapefile.jl/blob/21cc5c24ffd7f1f81dbeff18ce767e2090685589/test/shapelib_testcases/readme.txt), I think I generated them from running `shptest.c` in http://download.osgeo.org/shapelib/shapelib-1.3.0.tar.gz (archive of the other versions at http://download.osgeo.org/shapelib/).

**Comment by yeesian** [comment]
See also: https://github.com/OSGeo/shapelib/issues/15

**Comment by yeesian** [comment]
Possibly we should introduce new ones for https://github.com/GeospatialPython/pyshp/tree/b65a2b76968c5fe7698c3f7ea0c429e81a5f870a/shapefiles/test (discussion in https://github.com/JuliaGeo/Shapefile.jl/pull/40#issuecomment-673321884)
